app/src/main/res/layout/pic_of_day_app_widget.xml

```diff
@@ -46,7 +46,6 @@
         android:padding="@dimen/filter_padding"
         android:id="@+id/appwidget_image"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:contentDescription="@string/appwidget_img" />
```

Comment:
This content description seemed to me like removed instead of being added. Shouldn't we add it?

Reply:
I added back the comment description you mentioned in my most recent commit bccb40687fb844ef17d81d53a9fc373f1e27bfc8. Thank you for your help with this PR and I appreciate your feedback!